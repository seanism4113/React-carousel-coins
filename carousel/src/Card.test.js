import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";
import TEST_IMAGES from "./_testCommon";

it("renders without crashing", () => {
	render(<Card caption={TEST_IMAGES[0].caption} src={TEST_IMAGES[0].src} currNum={0} totalNum={TEST_IMAGES.length} />);
});

it("should match snapshot", () => {
	const { asFragment } = render(<Card caption={TEST_IMAGES[0].caption} src={TEST_IMAGES[0].src} currNum={0} totalNum={TEST_IMAGES.length} />);
	expect(asFragment()).toMatchSnapshot();
});
