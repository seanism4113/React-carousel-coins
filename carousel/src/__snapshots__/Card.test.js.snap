// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      testing image 1
    </h4>
    <img
      alt="testing image 1"
      class="Card-image"
      src="test1.com"
    />
    <small
      class="Card-small"
    >
      Image 0 of 3.
    </small>
  </div>
</DocumentFragment>
`;
